openshift/templates/jobs/backups/backup-cron.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: backup-mongodb
+  labels:
+    template: "backup-mongodb-cronjob"
+    cronjob: "backup-mongodb"
+spec:
+  schedule: "0 9 * * *" # 9am UTC = 1am PST
+  concurrencyPolicy: "Forbid"
+  successfulJobsHistoryLimit: 5
+  failedJobsHistoryLimit: 2
+  jobTemplate:
+    metadata:
+      labels:
+        template: "backup-mongodb-job"
+        cronjob: "backup-mongodb"
+    spec:
+      backoffLimit: 0
+      template:
+        metadata:
+          labels:
+            template: "backup-mongodb-job"
+            cronjob: "backup-mongodb"
+        spec:
+          containers:
+            - name: "backup-mongodb-cronjob"
+              image: >-
+                artifacts.developer.gov.bc.ca/bcgov-docker-local/backup-container-mongo:latest
```

Comment:
is :latest what we want.

Reply:
Yes, in my view that is better than choosing a specific sha value then not finding out for 6 months that there is an issue with that one, solved by latest.

Something crashes with the latest version, and we'll no quite quickly.